packages/common/src/content/edit.ts

```diff
@@ -104,6 +105,12 @@ export async function updateContent(
   // we are not attempting to handle "concurrent edit" conflict resolution
   // but this is where we would apply that sort of logic
   const modelToUpdate = mapper.entityToStore(content, model);
+
+  // prevent map from displaying when boundary is 'none'
+  const itemType = getProp(modelToUpdate, "item.properties.location.type");
+  modelToUpdate.item.properties.boundary =
+    itemType === "none" ? "none" : "item";
```

Comment:
`locationType` is more accurate

```suggestion
  const locationType = getProp(modelToUpdate, "item.properties.location.type");
  modelToUpdate.item.properties.boundary =
    locationType === "none" ? "none" : "item";
```